mailer.js

```diff
@@ -272,6 +273,43 @@ module.exports = function (log) {
     })
   }
 
+  Mailer.prototype.unblockCodeEmail = function (message) {
+    log.trace({ op: 'mailer.unblockCodeEmail', email: message.email, uid: message.uid })
+
+    var templateName = 'unblockCodeEmail'
```

Comment:
You can probably add `unblockCodeEmai` to the UTM [params map](https://github.com/mozilla/fxa-auth-mailer/pull/198/files#diff-3427c5e3d78d8d6550178ad46b9e0680R18). Not sure of a good name though? Maybe `new-unblock`?